config/initializers/new_gbl_settings_defaults_3_3.yml

```diff
@@ -0,0 +1,10 @@
+# New GeoBlacklight v3.3 Solr field mappings
+FIELDS:
+  :IDENTIFIER: 'dc_identifier_s'
+  :LANGUAGE: 'dc_language_s'
+  :LAYER_ID: 'layer_id_s'
```

Comment:
As @mejackreed suggested, this is redundant. This line can be removed and references to it changed to `WXS_IDENTIFIER`. Or vice versa, however I prefer WXS_IDENTIFIER as it's more descriptive.

Reply:
Sure thing! Just curious, but what does the WXS_ mean? Is it pertinent to all potential identifiers?